students/jiyeongkim/westagram/users/views.py

```diff
@@ -1,46 +1,64 @@
-from unicodedata import name
-from django.shortcuts import render
-
 import json
-import re
-
+import re 
+import bcrypt
+import jwt
 
 from django.http  import JsonResponse
 from django.views import View
 
+from westagram.settings import SECRET_KEY
 from users.models import User
 
 class SignUpView(View):
     def post(self, request) :
         try:
             data          = json.loads(request.body)
-            last_name     = data["last_name"]
-            first_name    = data["first_name"]
-            email         = data["email"]
-            password      = data["password"]
-            line          = data["line"]
-
-            EMAIL_CHECK = "^[a-zA-z0-9+_.]+@[a-zA-z0-9-.]+\.[a-zA-z0-9-.]+$"
-            PW_CHECK    = "^(?=.*[a-zA-Z])(?=.*\d)(?=.*[@$!%*#?&])[a-zA-Z\d@$!%*#?&]{8,}$"
+            last_name     = data['last_name']
+            first_name    = data['first_name']
+            email         = data['email']
+            password      = data['password']
+            line          = data['line']
+          
+            EMAIL_CHECK = '^[a-zA-z0-9+_.]+@[a-zA-z0-9-.]+\.[a-zA-z0-9-.]+$'
+            PW_CHECK    = '^(?=.*[a-zA-Z])(?=.*\d)(?=.*[@$!%*#?&])[a-zA-Z\d@$!%*#?&]{8,}$'
             
             if not re.match(EMAIL_CHECK, email) :
-                return JsonResponse({"Message": "INVALID_EMAIL"}, status=400)
+                return JsonResponse({'Message': 'INVALID_EMAIL'}, status=400)
             
             if not re.match(PW_CHECK, password) :
-                return JsonResponse({"Message": "INVALID_PASSWORD"}, status=400)
+                return JsonResponse({'Message': 'INVALID_PASSWORD'}, status=400)
```

Comment:
다음과 같은 곳이 있으면 전부 찾아서 고쳐주세요.
```suggestion
                return JsonResponse({'message': 'INVALID_PASSWORD'}, status=400)
```